### PR TITLE
Adds four more bounties

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: New bounties have been added for the Firefighter APLU mech, cat tails and the Cat/Liz o' Nine Tails weapons.
/:cl:

I added the following bounty items:
- Firefighter APLU for 18.000 dosh - 5.000 more than a Ripley APLU since it requires three times the amount of plasteel.
- Cat tail for 3000 since we already have a lizard tail bounty.
- Cat and Liz o' Nine Tails for 4000 as a variation on the tail bounties.

Edit: I removed the rebalance part so I can include it in a different PR.

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
can you remove japanotrasen citations

**Comment by Kagus** [comment]
Do we really even want these bounties? They're kinda the odd ones out due to being completely irreplaceable items, meaning that if they get lost/destroyed/sold they are permanently gone from the station until a new round. It ends up with either an empty bounty listing just sitting there and taking up space forever or some cognitively impaired cargo tech feverishly trying to break in because "I MUST MONEY" and they honestly have no idea what the actual value of the item even is.

And to those who say "conflict breeds gameplay", I have never seen the SS13 community fail to make their own conflicts at every opportunity.


Perhaps the items could be added to the cargo exports lists with fixed values, a la the lavaland relics, so that there's still a reason for exceptionally greedy quartermasters to grab the items, but it's not a blaring GET US THIS THING NOW OR BE FOREVER REMINDED OF IT bounty?

**Comment by Kromgar** [comment]
These are all irreplacable items. Why would anyone cash in on a 5k credit bounty?

**Comment by 81Denton** [comment]
@Kagus That‘s actually a really good point. When I get home, I‘ll remove the irreplacable item part from this PR and think about what to do with them.

**Comment by 81Denton** [comment]
Alright, I removed the part about irreplacable items for another PR. 
To me it seems like it's best to remove bounties for those items and instead add a fixed export price (with a note about it) or a separated bounty category for things like Lavaland loot. 
@Kagus are you fine with me quoting your post for that?

**Comment by Kagus** [comment]
@81Denton Absolutely, go right ahead.

**Comment by Wilchenx** [comment]
Not sure if its possible codewise, but you should make it so you need a combo of cat ears and tail.

File: code/modules/cargo/bounties/assistant.dm [hunk 1]
```diff
@@ -141,6 +141,26 @@
...
+	description = "Central Command has run out of heavy duty pipe cleaners. Can you ship over a cat tail to help us out?"
+	reward = 3000
+	wanted_types = list(/obj/item/organ/tail/cat)
+
+/datum/bounty/item/assistant/cat_tail_whip
+	name = "Cat o' Nine Tails"
+	description = "Commander Jackson is looking for yet another fine addition to her exotic weapons collection. She will reward you handsomely for a Cat o' Nine Tails."
+	reward = 4000
+	wanted_types = list(/obj/item/melee/chainofcommand/tailwhip/kitty)
+	exclude_types = list(/obj/item/melee/chainofcommand/tailwhip)
```
Comment (SpaceManiac) [inline code/modules/cargo/bounties/assistant.dm]: I believe the `exclude_types` here and on the Firefighter are unnecessary.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/modules/cargo/bounties/assistant.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/cargo/bounties/assistant.dm]: I'd at least keep the one on the Firefighter bounty so crew can't cash in with a much cheaper Ripley.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**